docs/software/android/usage.mdx

```diff
@@ -20,6 +20,10 @@ You will need a device with Meshtastic installed to go any further. See the [get
 
 To find devices to connect via Bluetooth click the button on the bottom right corner.
 
+:::note
+Scanning for BLE devices requires both location permission and for location services to be turned on during a scan. This is a [known issue](https://issuetracker.google.com/issues/37065090?pli=1).
```

Comment:
we can't say this is true in general. due to a bug, a minority of Android 8+ devices are not able to find new Bluetooth devices without location turned on. can you confirm this is what you are trying to address?